src/plugins/GUITestBase/src/base_dialogs/GTFileDialog.cpp

```diff
@@ -100,6 +104,8 @@ void GTFileDialogUtils::commonScenario() {
     GTGlobals::sleep(300);
     setViewMode(Detail);
     GTGlobals::sleep(300);
+    applyFilter();
```

Comment:
Sleep only if applyFilter is present (may be move sleep inside of applyFIlter)

Reply:
OK